README.md

```diff
@@ -364,6 +364,12 @@ An operation is about to be applied to the data. `source` will be `false` for op
 `doc.on('op', function(op, source) {...})`
 An operation was applied to the data. `source` will be `false` for ops received from the server and defaults to `true` for ops generated locally.
 
+`doc.on('before op batch'), function(op, source) {...})`
```

Comment:
As usual, naming is definitely up for debate. I'd have preferred to follow what was proposed in the [other PR](https://github.com/share/sharedb/pull/129), and rename the existing events to something like `op component`, but that would have been breaking.